docs/4.1/enterprise/ssh_fips.md

```diff
@@ -0,0 +1,150 @@
+| [SC-12 Cryptographic Key Establish and Management](https://nvd.nist.gov/800-53/Rev4/control/SC-12)  | Teleport initializes cryptographic keys that act as a Certificate Authority (CA) to further issue x509 and SSH certificates. SSH and x509 user certificates that are issued are signed by the CA and are (by default) short-lived. SSH host certificates are also signed by the CA and rotated automatically (a manual force rotation can also be performed). <br><br>Teleport Enterprise builds against a FIPS 140-2 compliant library (BoringCrypto) is available. In addition, when Teleport Enterprise is in FedRAMP/FIPS 140-2 mode, Teleport will only start and use FIPS 140-2 compliant cryptography. |
+
+
+
+
+
+Enterprise customers can download the custom FIPS package from the [Gravitational Dashboard](https://dashboard.gravitational.com/web/). 
+Look for `Linux 64-bit (FedRAMP/FIPS)`, with RPM and DEB packages easily available. 
+
+# Setup
+Customer can follow our [Enterprise Quickstart](quickstart-enterprise.md) for basic 
+instructions on how to setup Teleport Enterprise. You'll need to start with the Teleport 
+Enterprise FIPS Binary. 
+
+After downloading the binary tarball, run:
+
+```bsh
+$ tar -xzf teleport-ent-v4.1.4-linux-amd64-fips-bin.tar.gz
+$ cd teleport-ent
+$ sudo ./install 
+# This will copy Teleport Enterprise to /usr/local/bin.
+```
+
+## Configuration 
+
+### Teleport Auth Server
+Now, save the following configuration file as `/etc/teleport.yaml` on the _auth.example.com_:
+
+```yaml
+teleport:
+  auth_token: zw6C82kq7VEUSJeSDzuldWsxakql6jrTYmphxRQOlrATTGbLQoaIwEBo48o9
+  # Pre-defined tokens for adding new nodes to a cluster. Each token specifies
+  # the role a new node will be allowed to assume. The more secure way to
+  # add nodes is to use `ttl node add --ttl` command to generate auto-expiring
+  # tokens.
+  #
+  # We recommend to use tools like `pwgen` to generate sufficiently random
+  # tokens of 32+ byte length.
+  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
+  auth_servers: [ "10.1.1.10:3025" ]
+
+auth_service:
+  # enable the auth service:
+  enabled: true
+
+  tokens:
+  # this static token is used for other nodes to join this Teleport cluster
+  - proxy,node:zw6C82kq7VEUSJeSDzuldWsxakql6jrTYmphxRQOlrATTGbLQoaIwEBo48o9
+  # this token is used to establish trust with other Teleport clusters
+  - trusted_cluster:TaZff3DLbpsMZmIMhvEr7kulOgegjg7yyQNTS0q6UFWfsJ9N6rxVBjg6t7nw
+
+  # To Support FIPS local_auth needs to be turned off and a SSO connector is 
+  # required to log into Teleport. 
+  authentication:
+    # local_auth needs to be set to False in FIPS mode. 
+    local_auth: false
+    type: saml
+
+  # If using Proxy Mode, Teleport requires host key checks. 
```

Comment:
As this is also a requirement for running in FIPS mode, we should probably explicitly state that (as we do with local auth)